spec/requests/api/v2/providers/courses/create/age_range_in_years_spec.rb

```diff
@@ -85,7 +85,7 @@
       it "should return an error stating valid age ranges must be 4 years or greater" do
         expect(response).to have_http_status(:unprocessable_entity)
         expect(json_data.count).to eq 1
-        expect(response.body).to include "#{age_range_in_years} #{error_message}"
+        expect(response.body).to include "#{age_range_in_years.to_s.tr('_', ' ')} #{error_message}"
```

Comment:
What do you think about putting this into another `let` rather than doing the splitting in the expectation. Might be easier to read. e.g. `let(:expected_validation_age_range) { "1 to 15" }`. I prefer it when you can clearly see what you're expecting to happen rather than mentally executing the code. It is also exactly the code in the implementation so if there was an error in that code you wouldn't really see it in the test. e.g. if that returned ' ' for some reason it would return it in the code and the test so it would pass but wouldn't be what we actually wanted.